migrate/20250819_add_kubernetes_node_model.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:kubernetes_node) do
+      column :id, :uuid, primary_key: true
+      column :created_at, :timestamptz, null: false, default: Sequel::CURRENT_TIMESTAMP
```

Comment:
do we also need an updated_at column? In some models we have it (like `postgres_server`) and in some others we don't.

Reply:
maybe a question for @jeremyevans ?